src/coprocessor/dag/builder.rs

```diff
@@ -81,18 +70,14 @@ impl DAGBuilder {
                     //       It is undefined behavior if the source is unordered.
                     let descriptor = ed.get_aggregation();
                     BatchStreamAggregationExecutor::check_supported(&descriptor).map_err(|e| {
-                        Error::Other(box_err!(
-                            "Unable to use BatchStreamAggregationExecutor: {}",
-                            e
-                        ))
+                        Error::Other(box_err!("BatchStreamAggregationExecutor: {}", e))
                     })?;
                 }
                 ExecType::TypeLimit => {}
                 ExecType::TypeTopN => {
                     let descriptor = ed.get_topN();
-                    BatchTopNExecutor::check_supported(&descriptor).map_err(|e| {
-                        Error::Other(box_err!("Unable to use BatchTopNExecutor: {}", e))
-                    })?;
+                    BatchTopNExecutor::check_supported(&descriptor)
```

Comment:
Is it necessary to allocate for a common error?

Reply:
It is not a common error

Reply:
Then why people complains about the frequent logs?

Reply:
I doubt these logs (Enum / Set not supported) comes from previous versions. After #4788 very few cases would report this kind of error.

Reply:
Additionally one or few allocation is not worth mentioning for a Coprocessor request, which is very heavy already. For example, we have at least two allocations when scanning each row now. Thinking about how many allocations there will be for a table scan request.